src/styles/brackets_fonts.less

```diff
@@ -30,7 +30,11 @@
 /* SourceSansPro Regular */
 @font-face {
     font-family: 'SourceSansPro';
-    src: url('fonts/SourceSansPro/SourceSansPro-Regular.ttf');
+    src: local('Source Sans Pro Regular'),
+         url('fonts/SourceSansPro/SourceSansPro-Regular.eot?iefix') format('embedded-opentype'),
```

Comment:
Yea, it's probably not needed, but I included it because we're still trying to see if method helps #10680.